src/Installer.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @category    pimcore5-clipboard
+ * @date        25/10/2017 13:41
+ * @author      Monika Litwin <mlitwin@divante.pl>
+ * @copyright   Copyright (c) 2017 Divante Ltd. (https://divante.co)
+ */
+
+
+namespace Divante\ClipboardBundle;
+
+use Pimcore\Db;
+use Pimcore\Extension\Bundle\Installer\AbstractInstaller;
+use Pimcore\Extension\Bundle\Installer\Exception\InstallationException;
+use Pimcore\Tool\Admin;
+
+/**
+ * Class Installer
+ *
+ * @package Divante\ClipboardBundle
+ */
+class Installer extends AbstractInstaller
+{
+    /**
+     *
+     * @throws InstallationException
+     */
+    public function install()
+    {
+        $sql = file_get_contents(__DIR__ . '/Resources/sql/install.sql');
+        try {
+            Db::getConnection()->query($sql);
+        } catch (\Exception $ex) {
+            new InstallationException('An error occurred while installing the bundle', 0, $ex);
+        }
+    }
+
+    /**
+     *
+     * @throws InstallationException
+     */
+    public function uninstall()
+    {
+        $sql = file_get_contents(__DIR__ . '/Resources/sql/uninstall.sql');
+        try {
+            Db::getConnection()->query($sql);
+        } catch (\Exception $ex) {
+            new InstallationException('An error occurred while uninstalling the bundle', 0, $ex);
+        }
+    }
+
+    /**
+     * @return bool
+     */
+    public function isInstalled()
+    {
+        try {
```

Comment:
`\Pimcore\Bundle\EcommerceFrameworkBundle\Tools\Installer::isInstalled` - this approach will be better IMHO